snap-geometry-edits/src/main/java/com/esri/arcgismaps/sample/snapgeometryedits/screens/ButtonMenu.kt

```diff
@@ -107,18 +107,18 @@ fun ButtonMenu(
         ) {
             Icon(vector.vectorResource(R.drawable.undo), contentDescription = "Undo")
         }
-        IconButton(
-            enabled = mapViewModel.isSaveButtonEnabled.collectAsState().value,
-            onClick = { mapViewModel.stopEditor() }
-        ) {
-            Icon(vector.vectorResource(R.drawable.save), contentDescription = "Save")
-        }
         IconButton(
             enabled = mapViewModel.isDeleteButtonEnabled.collectAsState().value,
             onClick = { mapViewModel.deleteSelection() }
         ) {
             Icon(Icons.Filled.Delete, contentDescription = "Delete")
         }
+        IconButton(
+            enabled = mapViewModel.isSaveButtonEnabled.collectAsState().value,
+            onClick = { mapViewModel.stopEditor() }
+        ) {
+            Icon(vector.vectorResource(R.drawable.save), contentDescription = "Save")
+        }
         Row(
```

Comment:
Is there a reason why this has changed order?

Reply:
Given saving is the last thing we do when we're done editing, it made sense for it to be the right-most icon. Also, because the delete button can delete selected vertices, it can be used more often during editing.